DSharpPlus/Entities/DiscordMessageSticker.cs

```diff
@@ -24,31 +27,30 @@ public class DiscordMessageSticker : SnowflakeObject, IEquatable<DiscordMessageS
         /// Gets the Description of the sticker.
         /// </summary>
         [JsonProperty("description", NullValueHandling = NullValueHandling.Ignore)]
-        public string Description { get; set; }
+        public string Description { get; internal set; }
 
         /// <summary>
         /// Gets the list of tags for the sticker.
         /// </summary>
-        [JsonProperty("tags", NullValueHandling = NullValueHandling.Ignore)]
-        public string[] Tags { get; set; }
+        public string[] Tags => this._internalTags.Split(',');
```

Comment:
Might be a good idea to tag with `[JsonIgnore]` should sending ever came to be for whatever reason.